scripts/pre_commit_linter.py

```diff
@@ -1663,6 +1727,7 @@ def main():
     import_order_messages = _check_import_order(all_files)
     newline_messages = _check_newline_character(all_files)
     docstring_messages = _check_docstrings(all_files)
+    args_order_messages = _fetch_func(all_files)
```

Comment:
Your code is organized as `fetch_func` --calls--> `_check_args_order`, which means the high level call here is `fetch_func`, which is not descriptive of what this logic does. I think it would better to organize the code as `_check_args_order` --calls--> `fetch_func`, so that the high level function call is more descriptive. 
Also, that change would remove the call to `_check_args_order` in `fetch_func`, which makes `fetch_func` reusable in the future.